### PR TITLE
Let's actually hire someone

### PR DESCRIPTION
**Pull request by dabreegster**

### CODE REVIEW

**Comment by michaelkirk** [comment]
Nice! Once this is up you should consider posting in the georust discord.

**Comment by dabreegster** [comment]
> Nice! Once this is up you should consider posting in the georust discord.

Yep, it's finally happening -- Robin pulled some administrative magic. 100% going to advertise to georust!

File: book/src/project/hiring.md [hunk 1]
```diff
@@ -2,21 +2,49 @@
 
 A/B Street is hiring a Rust engineer!
 
+- **Timeline**: Full-time for 3 months, starting ASAP. Latest possible start
+  date is 1 June.
+- **Compensation**: £6,981 total
```
Comment (Robinlovelace) [inline book/src/project/hiring.md]: The final amount is not guaranteed having checked. I will update this.

File: book/src/project/hiring.md [hunk 2]
```diff
@@ -2,21 +2,49 @@
...
+- Must be UK-based with a UK bank account
```
Comment (Robinlovelace) [inline book/src/project/hiring.md:8]: :+1:

**Review by Robinlovelace — APPROVED** [review]

File: book/src/project/hiring.md (hunk 1 above)
Comment (dabreegster) [reply book/src/project/hiring.md]: Cool, thanks for the update (and breakdown!)

**Comment by dabreegster** [comment]
I'll interpret approval as mergeable!

**Pull request merged**